source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -49,42 +49,113 @@ public override CalificationsDto ExecuteWithResult()
+
+            // If loggedUser is the evaluated
+            // - Show auto-eval if not RFD
+            // - Show auto-eval & company if RFD
+
+            if (_loggedUser == _evaluatedUser)
+            {
+                return GetAutoEvaluation(evaluationDto, califications);
+            }
+
+            // If loggedUser is responsible
+            // - Show all
+
+            if (_loggedUser == evaluationDto.ResponsibleId)
+            {
+                return GetEvaluation(evaluationDto, califications);
+            }
+
+            // If loggedUser is evaluator
+            // - Show evluator eval
+
+            return GetEvaluatorEvaluation(evaluationDto, califications);
+        }
+
+        private CalificationsDto GetEvaluatorEvaluation(CalificationsEvaluationDto evaluationDto, List<EvaluationCalification> califications)
+        {
+            return new CalificationsDto()
+            {
+                View = UserView.Evaluation,
+                Evaluation = evaluationDto,
+                Califications = califications.Where(c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto).ToList()
+            };
+        }
+
+        private CalificationsDto GetEvaluation(CalificationsEvaluationDto evaluationDto, List<EvaluationCalification> califications)
+        {
+            return new CalificationsDto()
+            {
+                View = califications.Any(c => c.Owner == CalificationType.Company) ? UserView.Company : UserView.Responsible,
+                Evaluation = evaluationDto,
+                Califications = califications
+            };
+        }
+
+        private CalificationsDto GetAutoEvaluation(CalificationsEvaluationDto evaluationDto, List<EvaluationCalification> califications)
+        {
+            if (evaluationDto.ReadyForDevolution)
             {
                 return new CalificationsDto()
                 {
-                    View = califications.Any(c => c.Owner == CalificationType.Company) ? UserView.Company : UserView.Responsible,
+                    View = UserView.Auto,
                     Evaluation = evaluationDto,
-                    Califications = califications
+                    Califications = califications.Where(c => _evaluatedUser == c.EvaluatedEmployee && (c.Owner == CalificationType.Auto || c.Owner == CalificationType.Company)).ToList()
                 };
             }
-            else // Evaluator
+            else
             {
                 return new CalificationsDto()
                 {
-                    View = UserView.Evaluation,
+                    View = UserView.Auto,
                     Evaluation = evaluationDto,
-                    Califications = califications.Where(c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto).ToList()
+                    Califications = new List<EvaluationCalification>() { califications.Single(c => _evaluatedUser == c.EvaluatedEmployee && c.Owner == CalificationType.Auto) }
```

Comment:
I think that `Single` is not necessary here, you could use `Where` and `ToList` as in the rest of the cases.